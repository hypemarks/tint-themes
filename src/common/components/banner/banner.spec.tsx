import { render } from '@testing-library/react';
import React from 'react';
import Banner from './banner';

describe('Test <Banner /> Component', () => {
  it('should render snapshot', () => {
    const wrapper = render(<Banner banner={{ url: 'banner-url', height: 100 }} />);
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
