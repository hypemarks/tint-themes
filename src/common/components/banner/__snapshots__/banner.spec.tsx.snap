// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Banner /> Component should render snapshot 1`] = `
<DocumentFragment>
  <div
    class="banner"
    style="background-image: url(banner-url); height: 100px;"
  />
</DocumentFragment>
`;
